packages/pg-cursor/index.js

```diff
@@ -28,6 +28,9 @@ util.inherits(Cursor, EventEmitter)
 Cursor.prototype._ifNoData = function () {
   this.state = 'idle'
   this._shiftQueue()
+  if (this.connection) {
```

Comment:
Is it possible for `this.connection` to be falsy at this point, or was this just to match `_rowDescription`?

Reply:
I have no idea, because I am not very familiar with this project. I just follow what `_rowDescription` did to fix the problem and it works.